### ISSUE TITLE
Need to update package to v2.2.4

### ISSUE THREAD

**Issue by fvpDev** [Semantic-Org/Semantic-UI-Meteor#104]

**Comment by flemay** [comment]
Because it is not automated. I will try to update this week.

**Comment by fvpDev** [comment]
Oh...I thought it was at one point, no? Back with v1, no?

**Comment by purplecones** [comment]
yes, please do

**Comment by fvpDev** [comment]
Currently being updated to v2.2.4

Update: still waiting...

...never worked (not me updating, sry)

**Comment by purplecones** [comment]
I noticed 2.2.5 is not on https://cdnjs.com/libraries/semantic-ui. Is that what you are waiting for?

**Comment by philipmjohnson** [comment]
It would be really sweet if the Meteor integration would include the latest version of Semantic (2.2.4).

**Comment by eddiekollar** [comment]
Does a bump in the version number for semantic:ui and semantic:ui-data to 2.2.4 take care of this?

**Comment by jlukic** [comment]
I just released `2.2.5` , i'll look into fixing the meteor package. @flemay anything you can do on your end?

**Comment by Maxhodges** [comment]
how many months behind is the Official Meteor package typical trail the latest SUI?

**Comment by acemtp** [comment]
2.2.7 for semantic-ui and this package is 2.2.1 :-/

**Comment by vbgm** [comment]
Guys, semantic is the best ui library. And Meteor is the best framework. Please keep it going together. The update would be very appreciated.

**Comment by flemay** [comment]
@jlukic @Maxhodges @acemtp @vblagomir Sorry for the late reply. I could try to update to 2.2.7 (anyone could with https://github.com/flemay/Semantic-UI-Meteorize).

My main concern is when I developed the package it was for Meteor < 1.3 and I am really not up to date with the latest plugin system in Meteor. I know it has changed quite a bit. Prior updating the version, the package itself needs a refresh to reflect the latest changes in Meteor.

**Comment by Maxhodges** [comment]
i wish I could help, but I'm more of a product manager than a hard core Meteor dev. But I'm sure people on the Meteor forums would kindly answers questions you may have about packages.

**Comment by vbgm** [comment]
@flemay Thanks for your feedback! And great that there is the tool!

Regarding the package systems: over last half a year it was not clear with Meteor roadmap, but now it is. So far the standard meteor packages repository (Atmosphere) is alive and works ok, MDG pledges to maintain compatibility with new versions.  

There is general recommendation however to move meteor packages to npm (including the packages that are specifically made for meteor), so maybe someone one day could set it up, but not a big deal at the moment.

**Comment by flemay** [comment]
@vblagomir Good point you raised about Atmosphere and NPM. I was more targeting Meteor internal plugin API that I believe has changed considerably.

**Comment by vbgm** [comment]
@flemay I wrote couple of internal packages before and all of them work well so far on Meteor 1.4.*
You probably got this info, but just in case - changes in 1.3: https://guide.meteor.com/1.3-migration.html#packages
and overall packages guide: https://guide.meteor.com/writing-atmosphere-packages.html

**Comment by flemay** [comment]
I can update `Semantic-UI-Meteor-Data` to `2.2.6` but when I try publishing `Semantic-UI-Meteor` I get the following error:

```bash
node@7c513f35b763:/app$ make publish-ui
PACKAGE_PATH=dist/semantic-ui ./scripts/publish.sh
publishing dist/semantic-ui ...
generateSemanticUi: updating npm dependencies -- lodash, diff...
=> Errors while publishing:

While uploading build:
error: Error: ETIMEDOUT
```

I don't know if it is because my connection is too slow :(

**Comment by markudevelop** [comment]
@flemay any luck uploading it?

or any one has a guide here on how to switch to NPM/other ways to make it work outside of atmosphere?

**Comment by flemay** [comment]
@voidale @jlukic @vblagomir @acemtp @Maxhodges The packages have been updated to `2.2.6` but without being tested. 

Also I have dockerised https://github.com/flemay/Semantic-UI-Meteorize to ease the deployment process. Is anyone know if it is possible to `meteor publish` with a token file generated by `meteor login`? 

> You can pass METEOR_SESSION_FILE=token.json before meteor login to generate a login session token so you don’t have to share your login credentials with third-party service providers - https://docs.meteor.com/commandline.html

@jlukic I'd like to move the repo https://github.com/flemay/Semantic-UI-Meteorize to Semantic-Org but still does not have access.
